### ISSUE TITLE
No matching configuration found for 

### ISSUE THREAD

**Issue by laterdayi** [antfu/eslint-config#386]
### Describe the bug

No matching configuration found for 

![image](https://github.com/antfu/eslint-config/assets/122137647/18bc4069-9beb-473e-807a-82bf9cc53545)

### Reproduction

*

### System Info

```Shell
windows
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Dimava** [comment]
Please provide a reproduction repository published on Github, otherwise this issue will be immediately closed
> We could not identify whether it was a bug or a userland misconfiguration with the given info. Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue. Thanks.
> 
> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)


https://github.com/Dimava/antfu-eslint-config/tree/repro/386-works-fine-could-not-reproduce

**Comment by antfu** [comment]
We temporarily closed this due to the lack of enough information. We could not identify whether it was a bug or a userland misconfiguration with the given info.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)

**Comment by Dimava** [comment]
I've accidentally fount a reproduction today
```ts
export default antfu({...}, {
  files: ['**/*'],
  rules: {
    'antfu/consistent-list-newline': 'off',
  },
})
```
tl;dr don't use file globs without extension
Not a bug
